src/main/java/at/hannibal2/skyhanni/config/features/garden/laneswitch/LaneswitchConfig.java

```diff
@@ -0,0 +1,29 @@
+package at.hannibal2.skyhanni.config.features.garden.laneswitch;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import at.hannibal2.skyhanni.config.features.garden.visitor.TimerConfig;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.Accordion;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+
+public class LaneswitchConfig {
```

Comment:
Generally you capitalize new words in a class name:
```suggestion
public class LaneSwitchConfig {
```